packages/framework/src/Command/SortPhingTargetsCommand.php

```diff
@@ -31,31 +33,52 @@ protected function configure()
     /**
      * @param \Symfony\Component\Console\Input\InputInterface $input
      * @param \Symfony\Component\Console\Output\OutputInterface $output
+     * @return int
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
so in this case, dont we want to use also `declare(strict_types=1);` ?

Reply:
You're right, thanks for noticing!